integration-tests/docker/environment-configs/common

```diff
@@ -62,3 +62,4 @@ druid_zk_service_host=druid-zookeeper-kafka
 druid_auth_basic_common_maxSyncRetries=20
 druid_indexer_logs_directory=/shared/tasklogs
 druid_sql_enable=true
+druid_extensions_hadoopDependenciesDir=/shared/hadoop-dependencies
```

Comment:
Is this being used currently?

Reply:
Yes. The run_cluster.sh script runs pull-deps before starting docker containers. The pull-deps that is run will pull hadoop dependencies (hadoop-aws and hadoop-client) and put it into that path (/shared/hadoop-dependencies). When we run druid, we then set the druid.extensions.hadoopDependenciesDir to that same path (the line above), so that Druids can see the dependencies we pulled.